Core/include/Acts/Detector/DetectorComponents.hpp

```diff
@@ -36,11 +50,16 @@ class Portal;
 /// VolumeBounds derivative that is described by the given
 /// component.
 struct DetectorComponent {
-  using PortalContainer = std::map<unsigned int, std::shared_ptr<Portal>>;
+  /// The portal contaioner is used upstream to connect components
+  /// from different building blocks
+  using PortalContainer =
+      std::map<unsigned int, std::vector<std::shared_ptr<Portal>>>;
```

Comment:
This change is needed because the grid volumes have multiple portals as a container (and not in ACTS fusing one).